tchannel-core/src/main/java/com/uber/tchannel/utils/TChannelUtilities.java

```diff
@@ -14,15 +18,21 @@
 import java.util.Enumeration;
 
 public class TChannelUtilities {
+    private static final Logger logger = LoggerFactory.getLogger(TChannelUtilities.class);
+
     public static final ByteBuf emptyByteBuf = Unpooled.EMPTY_BUFFER;
 
-    public static final void PrintException(Throwable throwable) {
+    public static void setLogLevel(Level level) {
+        LogManager.getRootLogger().setLevel(level);
+    }
+
+    public static void PrintException(Throwable throwable) {
         StringWriter writer = new StringWriter();
         PrintWriter printWriter = new PrintWriter( writer );
         throwable.printStackTrace( printWriter );
```

Comment:
why do u need printWriter? just do printStackTrace if you want to output to console.